mono/metadata/mono-perfcounters.c

```diff
@@ -773,14 +802,14 @@ custom_category_help (SharedCategory* cat)
 }
 
 static const CounterDesc*
-get_counter_in_category (const CategoryDesc *desc, MonoString *counter)
+get_counter_in_category (const CategoryDesc *desc, mono_unwrapped_string_t counter)
 {
 	const CounterDesc *cdesc = &predef_counters [desc->first_counter];
 	const CounterDesc *end = &predef_counters [desc [1].first_counter];
-	for (; cdesc < end; ++cdesc) {
-		if (mono_string_compare_ascii (counter, cdesc->name) == 0)
+	for (; cdesc < end; ++cdesc)
```

Comment:
[Mono coding style](http://www.mono-project.com/community/contributing/coding-guidelines/) likes braces on the outer control structure when they're nested.

> good:﻿
> ```c
> if (a) {
>    if (b)
>        code ();
> }
> ```
> bad:
> ```c
> if (a)
>     if (b)
>         code ();
> ```

Reply:
Please put the braces around the for-loop body back.